pkg/runtime/executor.go

```diff
@@ -17,12 +17,27 @@
 package runtime
 
 import (
+	"fmt"
+
 	"go.uber.org/zap"
 
+	"github.com/accurics/terrascan/pkg/config"
 	iacProvider "github.com/accurics/terrascan/pkg/iac-providers"
 	"github.com/accurics/terrascan/pkg/notifications"
 	"github.com/accurics/terrascan/pkg/policy"
 	opa "github.com/accurics/terrascan/pkg/policy/opa"
+	"github.com/pelletier/go-toml"
+)
+
+const (
```

Comment:
I would suggest creating a new file `rules.go` and move these constants in that file.